lib/modules/manager/nuget/artifacts.ts

```diff
@@ -138,7 +138,11 @@ export async function updateArtifacts({
   }
 
   try {
-    if (updatedDeps.length === 0 && config.isLockFileMaintenance !== true) {
+    if (
+      updatedDeps.length === 0 &&
+      (config.updateType !== 'lockFileMaintenance' ||
+        config.isLockFileMaintenance)
```

Comment:
```suggestion
      (config.updateType !== 'lockFileMaintenance' &&
        config.isLockFileMaintenance !== true)
```

Shouldn't it be this?